docs/src/pages/customization/css-in-js.md

```diff
@@ -173,7 +173,7 @@ This `classes` object contains the name of the class names injected in the DOM.
 
 Some implementation details that might be interesting to being aware of:
 - It adds a `classes` property so you can override the injected class names from the outside.
-- It adds an `innerRef` property so you can get a reference to the wrapped component.
+- It adds an `innerRef` property so you can get a reference to the wrapped component. The usage of `innerRef` is identical to `ref`. The only difference is that it attaches to the wrapped component.
```

Comment:
Seems a bit redundant. The first sentence says exactly what it does.

Reply:
![capture d ecran 2018-01-06 a 19 32 46](https://user-images.githubusercontent.com/3165635/34642740-8342045a-f318-11e7-8ce0-eccaf655f043.png)

Reply:
So clarify the wording of the first sentence.